doc/api/process.md

```diff
@@ -745,7 +745,7 @@ process.exit(1);
 The shell that executed Node.js should see the exit code as `1`.
 
 It is important to note that calling `process.exit()` will force the process to
-exit as quickly as possible *even if there are still asynchronous operations 
```

Comment:
I assume you they have been inserted erroneously. Since other lines below have none I would prefer to have it "fixed" now. @jasnell do you have an opinion?